modules/apps/roles/roles-admin-web/src/main/resources/META-INF/resources/edit_role_assignments.jsp

```diff
@@ -33,6 +31,8 @@ EditRoleAssignmentsManagementToolbarDisplayContext editRoleAssignmentsManagement
 
 SearchContainer<?> searchContainer = editRoleAssignmentsManagementToolbarDisplayContext.getSearchContainer();
 
+String tabs2 = editRoleAssignmentsManagementToolbarDisplayContext.getTabs2();
```

Comment:
Please move this below the portletURL line

Reply:
https://github.com/liferay-commerce/liferay-portal/pull/4499/commits/ae4c5cbe1633495a270f2bd1e557e5a34d84e73a#diff-208df9741232b5d40768fbcb34d1f10fcdb02b90079bc3bf842fcb0bd1b2c5d5R36